REST/bash/Targets/ListeningLinuxTentacleStopTrustingOldOctopusServerCert.sh

```diff
@@ -0,0 +1,11 @@
+# Run on *each* Linux Listening Tentacle VM
+
+# Stop trusting the old certificate
+echo "Stop trusting the old Octopus Server certificate..."
+sudo ./Tentacle configure --instance Tentacle --remove-trust <old-thumbprint>

```

Comment:
Should the instance name `Tentacle` be a variable? Not all tentacles use the default named instance. They may have another name for it, meaning this and subsequent commands would fail